demos/gradle/gradlew

```diff
@@ -19,12 +20,17 @@ while [ -h "$PRG" ] ; do
         PRG=`dirname "$PRG"`"/$link"
     fi
 done
+# shellcheck disable=SC2006
```

Comment:
if you are going to add these comments, why not just change them to `$()`?

Reply:
Unless there is a way to skip the whole file without modifying it.

Reply:
This file was automatically generated, so I'd rather not actually try to fix all its numerous issues or I'll have to pretend I understand what this script is doing and test it :stuck_out_tongue: . I'll hard-code a skip for it.

Reply:
Done.